ui/apps/platform/src/Containers/Vulnerabilities/VulnerablityReporting/hooks/useDeleteDownloadModal.ts

```diff
@@ -0,0 +1,62 @@
+import useModal from 'hooks/useModal';
+import { useState } from 'react';
+import { deleteDownloadableReport } from 'services/ReportsService';
+import { getErrorMessage } from '../errorUtils';
```

Comment:
For consolidation later:

There's also a `utils/errorUtils` file which is similar to the local one imports here, and there is`getAxiosErrorMessage` helper function from` ui/apps/platform/src/utils/responseErrorUtils.ts` used in many places.